### DISCUSSION TITLE
Community support

### DISCUSSION THREAD

**Discussion by DawidWraga** [kumaaa-inc/shadow-panda#87] [General]
Hi, thanks for getting started on this awesome package. 

I am very interested in this project and would like to contribute so that most components are available within the next 2 weeks. 

I think it would be beneficial to communicate directly so that I can understand how to best help. 

Please could you provide me with any contact details or reach out to me on discord @dawidcodes. 

Thanks

**Comment by nanopx** [comment]
Hi @DawidWraga ,

Thank you for reaching out!

I saw your great feedback email, and will post a reply on discord today!

Thanks 😄

**Reply by nanopx** [reply]
Sent a request on discord 👍
